src/tools/perf/cuda/Makefile.am

```diff
@@ -7,8 +7,8 @@
 if HAVE_CUDA
 
 module_LTLIBRARIES               = libucx_perftest_cuda.la
-libucx_perftest_cuda_la_CPPFLAGS = $(BASE_CPPFLAGS) $(CUDA_CPPFLAGS)
-libucx_perftest_cuda_la_CFLAGS   = $(BASE_CFLAGS) $(CUDA_CFLAGS)
+libucx_perftest_cuda_la_CPPFLAGS = $(BASE_CPPFLAGS) $(CUDA_CPPFLAGS) $(LT_CFLAGS)
```

Comment:
no need to add to libucx_perftest_cuda_la_CPPFLAGS. need to add only to libucx_perftest_cuda_la_CFLAGS
same for all the rest

Reply:
Should I add it only to the CFLAGS in general? in all the Makefile.am files? @yosefe

Reply:
yes

Reply:
done